src/services/sam/access/types/accessResponseTypes.ts

```diff
@@ -0,0 +1,15 @@
+export interface GetKeyResponse {
+  uuid: string,
+  access: string,
+  name: string,
+  summary: string,
+  created_at: string,
+  last_used_at: string
+}
+
+export interface GetAccessibilityResponse{
```

Comment:
You may want to verify that the format of this response is correct. I could've sworn this endpoint only returns a bool

Reply:
it return a http_200_ok object which pass in a bool in `responses.http_200_ok(True)`

Reply:
should i still return a bool then?